install.js

```diff
@@ -10,14 +10,10 @@ const manifestURL =
   'https://s3.amazonaws.com/realm-clis/versions/cloud-prod/CURRENT';
 
 function fetchManifest() {
-  return new Promise(function(resolve, reject) {
-    axios.get(manifestURL, function(err, _, body) {
-      if (err) {
-        reject(err);
-        return;
-      }
-      resolve(JSON.parse(body));
-    });
+  return new Promise((resolve, reject) => {
```

Comment:
so `() => {}` style functions don't mesh well with particularly old versions of node (we've been burned by this before)... can we revert these types of changes (the intent still remains, just no "new style" function declarations)

Reply:
ahh good callout. will update 👍🏼